### ISSUE TITLE
Editing shortcode inserts nested shortcode

### ISSUE THREAD

**Issue by wernerkrauss** [sheadawson/silverstripe-shortcodable#11]
Hi,
on SS 3.1.10 and Shortcodable v 0.1.1 https://github.com/sheadawson/silverstripe-shortcodable/commit/5fbf1f86518045f6a84863c2da991bc83c62f687 when i click to edit an existing shortcode i see all data in the form (and heading: Edit Shortcode), but when i click on "insert shortcode" i get a nested shortcode:

before:  `[Webcam id=2 title="Foo"]`
after: `[Webcam id=2 t[Webcam id="2" title="Bar"]﻿itle="Foo"]`

**Comment by sheadawson** [comment]
Yeah I see, this needs fixing! I'm a bit limited on time at the moment but will have a look at it as soon as I can. If it helps in the mean time, you can select/highlight the existing shortcode text then click to edit, which works fine.

**Comment by wernerkrauss** [comment]
ping.... any improvement on this issue?

**Comment by sheadawson** [comment]
No movement on this sorry I just haven't had time to look at it. Hoping someone can send in a PR for this

**Comment by sheadawson** [comment]
The new placeholders will help with this but still need to fix shortcodes that don't have placeholders https://github.com/sheadawson/silverstripe-shortcodable/tree/2.0.2
